### PR TITLE
Update gh CLI steps in adding-a-new-ssh-key-to-your-github-account.md

### LINKED ISSUE #11721: update gh CLI steps in "Adding a new SSH key to your GitHub account"
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

### What part(s) of the article would you like to see updated?

I installed the `gh` CLI to add my ssh-key but was presented with this instead of a success message:
```
Welcome to GitHub CLI!

To authenticate, please run `gh auth login`
```

When I ran `gh auth login`, it prompted me to choose my preferred protocol and add my public key. Thus, I no longer needed the rest of the steps to add my ssh key with gh. If `gh` had already been authenticated, I would need the steps displayed. I think this nuance should be documented so people who are new to `gh` CLI are not confused and know they do not need to use the original gh steps by going down this path.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by michelleN**
### Why:
The GitHub CLI does not allow you to add an ssh key unless you are authenticated with your GitHub account. The `gh auth login` command not only authenticates you but also gives prompts to add your ssh key to your account. This PR amends the GitHub CLI portion with the `gh auth login` path. This may be useful to someone coming to this page for the first time who has not already installed the gh CLI and will do so as part of this flow and will then need to authenticate.

Closes #11721 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@michelleN 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md [hunk 1]
```diff
@@ -135,6 +135,13 @@ After adding a new SSH key to your account on {% ifversion ghae %}{% data variab
 
 {% data reusables.cli.cli-learn-more %}
 
+You'll need to authenticate the GitHub CLI with your GitHub account before being able to add an ssh-key. If you have not already done so, use the following command and follow the prompts to authenticate and add your ssh public key to your GitHub account.
+
+```shell
+gh auth login
+```
+
+If you are already authenticated, use the commands below.
```
Comment (lecoursen) [inline content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md:144]:
```suggestion
Before you can use the {% data variables.product.prodname_cli %} to add an SSH key to your account, you must authenticate to the {% data variables.product.prodname_cli %}. For more information, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login) in the {% data variables.product.prodname_cli %} documentation.
```

**Review by lecoursen — COMMENTED** [review]
Thanks @michelleN. Since the focus of this article is on adding an SSH key, let's mention the prerequisite of authenticating but link out for further details rather than including them here. I'll apply a suggestion to do that and then merge. 🎉

File: content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ After adding a new SSH key to your account on {% ifversion ghae %}{% data variab
 
 {% data reusables.cli.cli-learn-more %}
 
+Before you can use the {% data variables.product.prodname_cli %} to add an SSH key to your account, you must authenticate to the {% data variables.product.prodname_cli %}. For more information, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login) in the {% data variables.product.prodname_cli %} documentation.
```
Comment (lecoursen) [inline content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md:138]:
```suggestion
Before you can use the {% data variables.product.prodname_cli %} to add an SSH key to your account, you must authenticate to the {% data variables.product.prodname_cli %}. For more information, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login) in the {% data variables.product.prodname_cli %} documentation.

```

**Review by lecoursen — DISMISSED** [review]

**Review by lecoursen — APPROVED** [review]

**Pull request merged**